### DISCUSSION TITLE
aws binary is not working on linux build machine

### DISCUSSION THREAD

**Discussion by yod527** [codemagic-ci-cd/codemagic-docs#1085] [Q&A] [answered]
Faced an issue, where aws binary is not working on linux machine.

Build id: 63527dc635469db00bb41cc9

When using aws configure command in the script, it failed, however exactly the same configuration is working fine on macos build.

**Comment by icarusdust** [comment, marked as answer]
@yod527 Try SSHing into a builder machine and update aws cli tools for linux by running:

```
curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
unzip awscliv2.zip
sudo ./aws/install
```
Then run `/usr/local/bin/aws --version` to see if its upgraded to 2.8.4. If so, execute the same AWS login commands to see if it is still happening. Keep me updated.

**Reply by yod527** [reply]
ok, it helps - thanks
